app/controllers/collections_controller.rb

```diff
@@ -98,6 +94,13 @@ def download_xml
     send_data collection_download, type: content_type, disposition: "attachment; filename=#{concept_id}.#{download_format}"
   end
 
+  def delete_proposal
```

Comment:
This reads like it should be deleting something, but we are creating a delete proposal, so I think it should be `create_delete_proposal`